src/server/main_service.cc

```diff
@@ -1397,7 +1397,7 @@ void Service::DispatchMC(const MemcacheParser::Command& cmd, std::string_view va
       strcpy(cmd_name, "MGET");
       break;
     case MemcacheParser::FLUSHALL:
-      strcpy(cmd_name, "FLUSHDB");
+      strcpy(cmd_name, "FLUSHALL");
```

Comment:
why?

Reply:
Ah, good catch, forgot to follow up on this 😅 FLUSHDB is broken for some reason, will look at this